app/assets/stylesheets/static_pages.scss

```diff
@@ -0,0 +1,90 @@
+@import 'font-awesome-sprockets';
+@import 'font-awesome';
+@import 'bootstrap';
+@import 'color';
+
+html,body{
+background-image: image_url("544750.jpg");
+background-size: cover;
+background-repeat: no-repeat;
+height: 100%;
+font-family: 'Numans', sans-serif;
+}
+
+.container{
+height: 100%;
```

Comment:
toàn bộ phần khai báo style trong block đều lỗi indent nhé em
em sửa hết trong file này nhé